tools/deploy.sh

```
@@ -25,6 +25,21 @@ cp -r _maps/* $1/_maps/
+#иконок лист приезжает пустым, поэтому в деплой едут все .dmi всех деревьев кода.
+#config сюда не попадает намеренно: конфиг серверу подкладывают снаружи, а созданный
+#заранее каталог сломал бы тому же CI его собственный mkdir.
+find . \
+    -path ./.git -prune -o \
+    -path ./.claude -prune -o \
+    -path ./config -prune -o \
+    -path ./data -prune -o \
+    -path ./tools -prune -o \
+    -path "./$1" -prune -o \
+    -name "*.dmi" -print0 \
+    | xargs -0 -r cp --parents -t $1/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)